org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/form/AbstractForm.java

```diff
@@ -2348,7 +2354,17 @@ public void loadFromXml(Element root) {
         visit(v, IFormField.class);
         IFormField f = v.getField();
         if (f != null) {
-          f.loadFromXml(xField);
+          try {
+            success &= f.loadFromXml(xField);
+          }
+          catch (Exception e) {
+            LOG.warn("Could not load field {}", String.join(".", xmlFieldIds), e);
+            success = false; // error while loading value
+          }
+        }
+        else if (isXmlFieldWithStoredValue(xField)) {
```

Comment:
Does it really matter if the field has a value or not? Can we reliably detect that the xml field has a value?

Reply:
This also seems to be a behaviour change. Previously fields that could not be resolved were just ignored (regardless of having a value or not), but now we consider it to be an error. We should probably check with someone if that is ok. We may consider refactoring this a bit and overwrite the relevant parts in our own Form implementation if we have stricter requirements to what is a successful load and what is not.